assignment4/solvers/policy_iteration.py

```diff
@@ -51,10 +51,6 @@ def step(self):
                 self._policy_stable = False
             self._policy[s] = np.eye(self._env.nA)[best_a]
 
```

Comment:
I'm not sure this code is needed. self._policy_stable will get set to False any time a state changes, which seems sufficient to tell if it converged. The "delta" in this case seems to only track the largest change across all the states, but we should care more about either the total change across all states, or else simply if any state changed at all. I might be missing something here though...

Reply:
Yeah, it's possible this came from a lot of hacking at the code to figure things out as I went. I think I wanted to have a softer check for convergence as well as a guaranteed minimum number of steps. I wonder if this should be commented out with a note that it might help or just removed. My main concern is if it does help someone and then it's taken out... hrm.

Reply:
Yeah, I'm actually not sure this is the right call anymore. Closing the PR for now